.circleci/config.yml

```
@@ -95,7 +95,7 @@ jobs:
             package="${CIRCLE_JOB#build_}"
             package="${package//_/-}"
 
-            cd "packages/@sfx/${package}"
+            cd "packages/@groupby/${package}"
```

If you follow the suggestion above and drop `elements` from all the job names, this and similar lines will change to the following:

```suggestion
            cd "packages/@groupby/elements-${package}"
```